AdobeStockImageAdminUi/view/adminhtml/web/js/mediaGallery/grid/columns/image/licenseActions.js

```diff
@@ -97,9 +97,7 @@ define([
                     this.imageModel().reloadGrid();
                     this.imageModel().addMessage('success', $.mage.__('The image has been licensed.'));
                 }.bind(this)).fail(function (error) {
-                    if (error) {
-                        this.imageModel().addMessage('error', error);
-                    }
+                    this.imageModel().addMessage('error', error);
```

Comment:
The message should not be shown if there is no error (in case the confirmation was closed by the user)

Also can you please:
remove the empty return from licenseProcess function (it should always return promise)

Reply:
ok